redaxo/src/core/lib/api_function.php

```diff
@@ -81,6 +81,16 @@ public static function factory()
 
         $api = rex_request(self::REQ_CALL_PARAM, 'string');
 
+        if (empty($api)) {
+            $messageBody = file_get_contents('php://input');
```

Comment:
side-note: ggf. Würde sich diese logik auch in rex_request unterbringen lassen, weil generisch (raw post lesen und parsen)?

Reply:
Der Part soll ganz entfallen: https://github.com/redaxo/redaxo/pull/4554/files#r590203789